templates/docs/examples/patterns/contextual-menu/overlapping.html

```diff
@@ -0,0 +1,26 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Contextual menu / Overlapping{% endblock %}
+
+{% block standalone_css %}patterns_contextual-menu{% endblock %}
+
+{% block content %}
+<span class="p-contextual-menu--left is-overlapping">
+  <button class="p-button--positive p-contextual-menu__toggle has-icon" aria-controls="menu-3" aria-expanded="false" aria-haspopup="true"><i class="p-icon--chevron-down is-light p-contextual-menu__indicator"></i><span>Take action</span></button>
+  <span class="p-contextual-menu__dropdown" id="menu-3" aria-hidden="true">
+    <span class="p-contextual-menu__group">
+      <a href="#" class="p-contextual-menu__link">Commission</a>
+      <a href="#" class="p-contextual-menu__link">Aquire</a>
+      <a href="#" class="p-contextual-menu__link">Deploy</a>
+    </span>
+    <span class="p-contextual-menu__group">
+      <a href="#" class="p-contextual-menu__link">Test harware</a>
```

Comment:
Not relevant to the UX, but typo on Hardware

Reply:
good catch - fixed